05LessonFive/LessonFive.js

```diff
@@ -15,6 +25,19 @@ module.exports = function() {
     // divisible by 3 are replaced by the string "fizz", numbers divisible by 5
     // are replaced by the string "buzz", and the numbers divisble by both 3 and
     // 5 are replaced by fizzbuzz
-
-  }
+	  var array = [];
+	  	  for(var i = 0; i < 100; i++){
+	  	       if (i%3 ===0 && i%5===0) {
+	  	  			  array.push("fizzbuzz");
+	  	  		  } else if (i%3 === 0){
+	  	  			  array.push("fizz");
+	  	  		  } else if (i%5 === 0){
+	  	  			  array.push("buzz");
+	  	  		  } else {
+	  			  array.push(i);
+	  	  	      }
+			  }
+	  		  
+	  	  return array; 	
+}
```

Comment:
Hey Joseph,

Didn't see these emails, thanks for the help and tips, I will fix this. You
stay up late.

On Tue, Oct 13, 2015 at 3:49 AM Joseph McCullough <notifications@github.com>
wrote:

> In 05LessonFive/LessonFive.js
> <https://github.com/AustinCodingAcademy/HTMLIntermediateSection2Tests/pull/6#discussion_r41841172>
> :
>
> > -  }
> > +	  var array = [];
> > +	  	  for(var i = 0; i < 100; i++){
> > +	  	       if (i%3 ===0 && i%5===0) {
> > +	  	  			  array.push("fizzbuzz");
> > +	  	  		  } else if (i%3 === 0){
> > +	  	  			  array.push("fizz");
> > +	  	  		  } else if (i%5 === 0){
> > +	  	  			  array.push("buzz");
> > +	  	  		  } else {
> > +	  			  array.push(i);
> > +	  	  	      }
> > +			  }
> > +	  		
> > +	  	  return array; 	
> > +}
>
> Looks good. [image: :+1:] Please fix the indention here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AustinCodingAcademy/HTMLIntermediateSection2Tests/pull/6/files#r41841172>
> .
>
-- 
Peter Arevalo